index.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
 
 module.exports = {
-  extends: ['prettier'],
-  parserOptions: {
-    ecmaVersion: 9
-  },
-  env: {
-    es6: true,
-    browser: true, // browser global variables.
-    node: true, // Node.js global variables and Node.js-specific rules.
-    commonjs: true, // defines require() and define() as global variables as per the amd spec.
-    mocha: true, // adds all of the Mocha testing global variables.
-    jquery: true // jquery global variables.
-  }
+    extends: ['prettier'],
+    parserOptions: {
```

Comment:
I think es2021 is the right version to go with here. Looking at the Node, V8, and ECMAScript docs, I think that's the highest implemented version but I could be wrong so let me know what you think.